introduction.md

```diff
@@ -1,2 +1,6 @@
 Hi! This is an introduction.
-I like iOS
\ No newline at end of file
+I like iOS
+* android
+* java
+* git
+* freeme
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?